frontend/src/routes/Governance/policies/policy-details/PolicyDetailsResults.tsx

```diff
@@ -258,10 +260,16 @@ export default function PolicyDetailsResults() {
       {
         header: t('Last report'),
         sort: 'timestamp',
-        cell: (item: ResultsTableData) =>
-          item.timestamp ? moment(item.timestamp, 'YYYY-MM-DDTHH:mm:ssZ').fromNow() : '-',
-        exportContent: (item: ResultsTableData) =>
-          item.timestamp ? moment(item.timestamp, 'YYYY-MM-DDTHH:mm:ssZ').fromNow() : '-',
+        cell: (item: ResultsTableData) => {
```

Comment:
Since `exportContext` uses the same logic, could we extract it into a separate functionAlso. In addition, that way is good for sonarCloud lint